tests/e2e/operationstorm.go

```diff
@@ -208,4 +211,84 @@ var _ = utils.SIGDescribe("[csi-block-vanilla] Volume Operations Storm", func()
 			}
 		}
 	})
+
+	ginkgo.It("Delete namespace to confirm all volumes and pods are deleted", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		ginkgo.By(fmt.Sprintf("Running test with VOLUME_OPS_SCALE: %v", volumeOpsScale))
+		ginkgo.By("Creating Storage Class")
+
+		// decide which test setup is available to run
+		if vanillaCluster {
+			ginkgo.By("CNS_TEST: Running for vanilla k8s setup")
+			scParameters = nil
+			storagePolicyName = ""
+		} else {
```

Comment:
This test case is for only vanilla, right ?
In that why do we need else {}

Reply:
Test execution is extended to WCP and GC as well